website/apps/homepage/views.py

```diff
@@ -25,6 +33,12 @@ def get_quote(self):
             )
 
     def get_context_data(self):
+        if self.request.mission.memorial:
+            people, more_people = mission_people(self.request)
```

Comment:
Perhaps `significant_people` and `other_people`? It's not just `mission_ops_people` in the second list IIRC.